README.md

```diff
@@ -69,10 +69,13 @@ Specifically, the files needed are `ingest/results/metadata.tsv` and `ingest/res
 Run full genome builds with the following command.
 
 ``` bash
-nextstrain build . --snakefile Snakefile.genome
+nextstrain build . --snakefile Snakefile.genome --config local_ingest=True ingest_source=ncbi
 ```
 
-Currently this is only set up for the "h5n1-cattle-outbreak" build, and restricts the build to a set of strains where we think there's no reassortment (`config/include_strains_h5n1-cattle-outbreak.txt`). Output files will be placed in `results/h5n1-cattle-outbreak/genome`. See `Snakefile.genome` for more details.
+Currently this is only set up for the "h5n1-cattle-outbreak" build using locally ingested NCBI data. and The build is restricted to a set of strains where we think there's no reassortment, with outgroups
```

Comment:
```suggestion
Currently this is only set up for the "h5n1-cattle-outbreak" build using locally ingested NCBI data, and the build is restricted to a set of strains where we think there's no reassortment, with outgroups
```

Reply:
Included in https://github.com/nextstrain/avian-flu/pull/40/commits/eb5e068ef2d4260b01c7ac49fa805b539fb05e83